tests/unit/setup/test_install.py

```diff
@@ -193,6 +201,14 @@ def test_egg(self, tempdir):
                 salt_generated_version_file_path.is_file()
             ), "{} is not a file".format(salt_generated_version_file_path)
 
+    # On python 3.5 Windows sdist fails with encoding errors. This is resolved
+    # in later versions.
+    @skipIf(
+        salt.utils.platform.is_windows()
+        and sys.version_info > (3,)
```

Comment:
This one isn't really necessary, we no longer officially support Py2 on the master branch.
Not a merge blocker though.

Reply:
It's skipping on python 3.5 but which is what we're running on windows right now.

Reply:
I only meant `sys.version_info > (3,)`